src/main/kotlin/org/rust/ide/formatter/processors/RsTrailingCommaFormatProcessor.kt

```diff
@@ -97,10 +96,11 @@ fun CommaList.addTrailingCommaForElement(list: PsiElement): Boolean {
     return true
 }
 
-fun CommaList.isLastElement(list: PsiElement, element: PsiElement): Boolean {
+fun CommaList.isOnSameLineAsLastElement(list: PsiElement, element: PsiElement, document: Document): Boolean {
```

Comment:
Let's make it `private`

Reply:
Looks like passing `document` is redundant since it can be accessed via `element.containingFile.document`

Reply:
Cannot be private.

Reply:
Oh, right. I didn't notice it's a different file